pkg/kubernetes/manifest/manifest.go

```diff
@@ -133,6 +133,11 @@ func (m Metadata) Labels() map[string]interface{} {
 	return m["labels"].(map[string]interface{})
 }
 
+// SetLabels sets the labels of a manifest
+func (m Metadata) SetLabels(labels map[string]interface{}) {
```

Comment:
No need for that, a `map[string]interface{}` is a pointer type so you can just assign values on the return value of `Labels()`